npm/.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+	"useTabs": true
```

Comment:
Why is this preferred over 2 spaces?

Reply:
[Tabs are significantly better than spaces wrt a11y](https://twitter.com/DasSurma/status/1553634641491509248) :)